src/Configuration/Defaults/CommonRules.php

```diff
@@ -330,5 +331,6 @@ class CommonRules extends Rules
             "anonymous_functions_opening_brace" => "same_line",
         ],
         LowercaseKeywordsFixer::class => true,
+        NoMultilineWhitespaceAroundDoubleArrowFixer::class => true,
```

Comment:
It may be worth documenting what else we're bringing on when including this fixer.

According to documentation [here](https://cs.symfony.com/doc/rules/array_notation/no_multiline_whitespace_around_double_arrow.html), this fixer deals with whitespace around double arrows in arrays. Their example is: 

```php
// 👎
$a = array(1

=> 2);

// 👍
$a = array(1 => 2);
```

The example we're documenting here is an edge case of the above.
Another thing it's doing is messing with arrow functions: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/5908 / https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/6002
This might or might not be what we want - but if it is, we could at least document it.

Reply:
added more test cases